css/style.css

```diff
@@ -87,3 +88,161 @@ section.headline {
 .socials .icon:hover {
   cursor: pointer;
 }
+
```

Comment:
- [ ] It would be so much better if the portfolio on the mobile version doesn't have a scroll bar at the bottom. This will make the app looks bad. Please fix this.

![Untitled](https://user-images.githubusercontent.com/64173189/133855798-8bfbdc87-7a44-4f72-afe9-1e8ecb6eebdc.png)

Reply:
Fixed.